practice-app/website/api/event.py

```diff
@@ -2,11 +2,21 @@
 from ..models import Event
 from .. import db
 import requests
+from ..views import get_sport_names
 from flasgger.utils import swag_from
 from sqlalchemy import exc
 events = Blueprint('events', __name__)
 
-API_KEY = '<api key>'
+API_KEY = 'AIzaSyB918Ru6ZsU1P3OPc-IaE_lRLgjNM1Suyk'
```

Comment:
Please put a placeholder here, and for other api_keys

Reply:
I have put a placeholder.